joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -39,6 +39,20 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
     Seq(Ast(literalNode(_literalNode, code(_literalNode), nodeTypeFullName(_literalNode))))
   }
 
+  protected def astForExpressionNode(expressionNodeInfo: DotNetNodeInfo): Seq[Ast] = {
```

Comment:
I wonder if we should rename this to `astForOperand` if that's the only use case for this special case method

Reply:
Also, there exists `astForExpression`, so that's why I'm pretty keen for the name to be a bit more specific to reduce ambiquity